packages/file_selector/file_selector_windows/lib/src/dart_file_selector_api.dart

```diff
@@ -0,0 +1,328 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:file_selector_windows/src/messages.g.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:win32/win32.dart';
+
+import 'dart_file_dialog.dart';
+import 'dart_file_open_dialog_api.dart';
+import 'dart_shell_item_api.dart';
+
+/// Dart native implementation of FileSelectorAPI
```

Comment:
This doesn't match the `extends`, which makes this comment very confusing to me.

Reply:
Hi Stuart, it was an error, we updated the hierarchy and the comments.